run_project_tests.py

```diff
@@ -321,18 +337,16 @@ def run_test_inprocess(testdir):
         os.chdir(old_cwd)
     return max(returncode_test, returncode_benchmark), mystdout.getvalue(), mystderr.getvalue(), test_log
 
-def parse_test_args(testdir):
-    args = []
-    try:
-        with open(os.path.join(testdir, 'test_args.txt'), 'r') as f:
-            content = f.read()
-            try:
-                args = literal_eval(content)
-            except Exception:
-                raise Exception('Malformed test_args file.')
-            args = stringlistify(args)
-    except FileNotFoundError:
-        pass
+def parse_test_args(testdir: str) -> typing.List[str]:
+    args = []  # type: typing.List[str]
+    file = Path(testdir, 'test_args.txt')
```

Comment:
linters are probably going to complain about shadowing the `file` builtin.

Reply:
`file` is not a builtin in Python 3.